### PR TITLE
Create an Author class

### PR DESCRIPTION
**Pull request by jaflih**
- Created unit tests for the Author class 
- Create an Author class with an association to the Item class (in a separate .rb file).
- Setup all Author class properties in the constructor method.

### CODE REVIEW

File: application/author.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  # def add_item(item)
+  #  @items << item
+  # end
```
Comment (Ginohmk) [inline application/author.rb:17]: It is advisable to remove commented code so as to make your code cleaner.

File: spec/author_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  # it 'Check addItem()' do
+  #  expect(@author.items).to eq []
+  #  item = 'item' # Item.new('genre', @author, 'source', 'label', '1999/02/02', true)
+  #  @author.add_item(item)
+  #  expect(@author.items).to eq [item]
+  #  item2 = 'item2' # Item.new('genre', @author, 'source', 'label', '1999/02/02', true)
+  #  @author.add_item(item2)
+  #  expect(@author.items).to eq [item, item2]
+  # end
```
Comment (Ginohmk) [inline spec/author_spec.rb:40]: - [ ]  Same thing here, if the commented code is not needed, you can take it out

**Review by Ginohmk — APPROVED** [review]
You have properly arranged your code files into folders, giving them a clean structure. There are some optional things to look at below.

Other than that this pull request is approved    image

File: application/author.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  attr_accessor :first_name, :last_name
+  attr_reader :id, :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+end
```
Comment (Ginohmk) [inline application/author.rb:11]: You are missing a method add_item which should 
image

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Looking at the requirements, the Author class should have a method called add_item

cross-check the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md) under member #3

**Comment by jaflih** [comment]
Hi @Ginohmk ,

Thank you for the review.
I didn't start the [task implemented methods]( https://github.com/jaflih/catalog_of_my_things/issues/26)

This PR concerns only: [this task](https://github.com/jaflih/catalog_of_my_things/issues/23) and [this task](https://github.com/jaflih/catalog_of_my_things/issues/25)

**Comment by Ginohmk** [comment]
Alright, @jaflih . I felt like the class author, would require you to add all its requirements. Alright, I will approve it. Thanks for reaching out.

**Review by Ginohmk — APPROVED** [review]

**Pull request merged**